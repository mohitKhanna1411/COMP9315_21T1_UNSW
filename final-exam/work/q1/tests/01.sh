./pages  3  4  "1,1,2,2" <<_Q_
?,?,?
a,b,c
a,b,?,
a,?,c
quit
_Q_
