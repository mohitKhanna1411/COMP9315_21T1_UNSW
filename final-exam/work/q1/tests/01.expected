CV = < (1,0) (1,1) (2,0) (2,1) >
query> q[1] = ??
q[2] = ??
q[3] = ??
Pages: 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = 11101011 01100101 01010001 10111010
q[3] = 01101100 01000111 11011011 00100111
Pages: 9
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = 11101011 01100101 01010001 10111010
q[3] = ??
Pages: 9
query> q[1] = 00101111 11100001 00011010 01110001
q[2] = ??
q[3] = 01101100 01000111 11011011 00100111
Pages: 1 5 9 13
query> 
